pages/api/graphql.js

```diff
@@ -0,0 +1,30 @@
+import { ApolloServer, gql } from 'apollo-server-micro';
+import dbConnect from 'utils/dbConnect';
+
+import User from 'models/User';
+import Space from 'models/Spaces';
+import { typeDefs, resolvers } from 'utils/schema';
+const apolloServer = new ApolloServer({ typeDefs, resolvers });
+
+const startServer = apolloServer.start();
+
+export default async function handler(req, res) {
+  res.setHeader('Access-Control-Allow-Credentials', 'true');
+  res.setHeader('Access-Control-Allow-Origin', 'https://studio.apollographql.com');
+  res.setHeader('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+  if (req.method === 'OPTIONS') {
+    res.end();
+    return false;
```

Comment:
btw why are you returning false? Was this in a tutorial? I'm unsure of why you need it here.